01-html/04-nytimes.html

```diff
@@ -1,3 +1,1120 @@
+        <li id="stocks" class="onmouseover">Wall Street API</li>
+      </ul>
+    </nav>
+    <hr>
+    <nav class="inline-block no-text-decoration position-fixed">
+      <ul>
+        <li><a href="http://www.nytimes.com/pages/world/index.html">World</a></li>
+        <li><a href="http://www.nytimes.com/section/us">U.S.</a></li>
+        <li><a href="http://www.nytimes.com/pages/politics/index.html">Politics</a></li>
+        <li><a href="http://www.nytimes.com/section/nyregion">N.Y.</a></li>
+        <li><a href="http://www.nytimes.com/pages/business/index.html">Business</a></li>
+        <li><a href="http://www.nytimes.com/pages/opinion/index.html">Opinion</a></li>
+        <li><a href="http://www.nytimes.com/pages/technology/index.html">Tech</a></li>
+        <li><a href="http://www.nytimes.com/section/science">Science</a></li>
+        <li><a href="http://www.nytimes.com/pages/health/index.html">Health</a></li>
+        <li><a href="http://www.nytimes.com/pages/sports/index.html">Sports</a></li>
+        <li><a href="http://www.nytimes.com/pages/arts/index.html">Arts</a></li>
+        <li><a href="http://www.nytimes.com/section/fashion">Style</a></li>
+        <li><a href="http://www.nytimes.com/pages/dining/index.html">Food</a></li>
+        <li><a href="http://www.nytimes.com/section/travel">Travel</a></li>
+        <li><a href="http://www.nytimes.com/section/magazine">Magazine</a></li>
+        <li><a href="http://www.nytimes.com/section/t-magazine">T Magazine</a></li>
+        <li><a href="http://www.nytimes.com/pages/realestate/index.html">Real Estate</a></li>
+        <li id="change-font"><a href="#side-full-menu">All</a></li>
+      </ul>
+    </nav>
+  </header>
+  <hr>
+  <aside id="left-side">
+    <article>
+      <hgroup>
+        <h4 id="sub-heading">
+          <img src="http://thumb9.shutterstock.com/display_pic_with_logo/686785/364157066/stock-photo-ballot-box-with-american-flag-d-illustration-364157066.jpg" alt="3d-ballot-box" height="30">ELECTION 2016</h4>
+        <h2 id="large-article-heading">Labor Leaders Fear Trump's Appeal With Rank and File</h2>
+      </hgroup>
+      <address class="byline">By NOAM SCHEIBER</address>
+      <p>Labor leaders fear that Donald J. Trump may draw a large number of union voters, possibly improving Republican chances in a November general election.
+        <a href="http://www.nytimes.com/2016/01/30/business/donald-trump-unions.html?hp&target=comments#commentsContainer">
+          <img src="http://www.homegridforum.org/images/home/slider_images/slide_04_blue_rectangle.png" alt="blue-rectangle" height="15">718 Comments</a>
+      </p>
+      <h5 class="related-articles">MORE COVERAGE</h5>
+      <ul>
+        <strong>
+          <li><a href="http://www.nytimes.com/2016/01/30/us/politics/22-clinton-emails-deemed-too-classified-to-be-made-public.html">22 Clinton Emails Deemed Too Secret to Be Released</a>
```

Comment:
Style elements should be used inline, if at all. For example, if you wanted to **emphasize something mid-sentence**, that's a good time to use `strong`. Otherwise, you'll style it by attaching a class and adding CSS.